download-vector-tiles-to-local-cache/src/main/java/com/esri/arcgismaps/sample/downloadvectortilestolocalcache/MainActivity.kt

```diff
@@ -45,14 +43,15 @@ import com.arcgismaps.tasks.exportvectortiles.ExportVectorTilesResult
 import com.arcgismaps.tasks.exportvectortiles.ExportVectorTilesTask
 import com.esri.arcgismaps.sample.downloadvectortilestolocalcache.databinding.ActivityMainBinding
 import com.esri.arcgismaps.sample.downloadvectortilestolocalcache.databinding.ProgressDialogLayoutBinding
+import com.google.android.material.button.MaterialButton
 import com.google.android.material.snackbar.Snackbar
 import kotlinx.coroutines.launch
 import java.io.File
 
 class MainActivity : AppCompatActivity() {
 
     private val downloadArea: Graphic = Graphic()
-    private var dialog: AlertDialog? = null
+    private var dialog: androidx.appcompat.app.AlertDialog? = null
```

Comment:
Can you add the import statement here 
```kotlin
import androidx.appcompat.app.AlertDialog
```
```suggestion
    private var dialog: AlertDialog? = null
```

Reply:
Sorry i thought I changed it, to add the import and not use it that way.